src/unix/notbsd/linux/other/b32/x86.rs

```diff
@@ -18,3 +18,53 @@ pub const SO_SNDTIMEO: ::c_int = 21;
 
 pub const FIOCLEX: ::c_ulong = 0x5451;
 pub const FIONBIO: ::c_ulong = 0x5421;
+
+type greg_t = ::c_int;
+
+const NREG : usize = 19;
+
+type gregset_t = [greg_t; NREG];
+
+s! {
+
+    pub struct _libc_fpreg {
+        significand: [::c_ushort; 4],
+        exponent: ::c_ushort,
+    }
+
+    pub struct _libc_fpstate {
+        cw: ::c_ulong,
+        sw: ::c_ulong,
+        tag: ::c_ulong,
+        ipoff: ::c_ulong,
+        cssel: ::c_ulong,
+        dataoff: ::c_ulong,
+        datasel: ::c_ulong,
+        _st: [_libc_fpreg; 8],
+        status: ::c_ulong,
+    }
```

Comment:
Since these are just private structures with all private fields, can they be left out for now? They can likely just be represented via `__data: [usize; AMT]` below somehow